### ISSUE TITLE
aowow_quests is empty

### ISSUE THREAD

**Issue by TheWinchesters** [Sarjuuk/aowow#157]
I do not know why, the aowow_quests table is empty.
Everything else is ok.

All other tables are ok, except this one.
Website is showing everything except quests ofcourse.
I have tried reuploading DBFilesClient, syncing tables related to quests and still nothing.

I have this, but it looks like it does nothing:
SqlGen::generate() - filling aowow_quests with data
[OK]     - subscript 'quests' returned sucessfully

What can I do?

**Comment by TheWinchesters** [comment]
When I try to run the select that quests.func.php has, I have this error:
Error Code: 1054. Unknown column 'qtl2.OfferRewardText' in 'field list'

I may think that the problem is that TC made some changes about quests_*_locale
https://github.com/TrinityCore/TrinityCore/commit/2f5403d4af3a90a6e51a377e05a55a0d197afb4d

**Comment by Sarjuuk** [comment]
nooo ... not again ... :sob:

**Comment by TheWinchesters** [comment]
Is there anything I can help you with?
I could try to make the select query work.

**Comment by TheWinchesters** [comment]
I made it work (at least for me).
I have just modified some things in the select query to match the new TrinityCore style for quest_*_locale tables.

In quests.func.php file:
I have replaced line 83 and 84 with:
```
IFNULL(qor.RewardText, ""),         IFNULL(qorl2.RewardText, ""),   	 IFNULL(qorl3.RewardText, ""),   	  IFNULL(qorl4.RewardText, ""),   	   IFNULL(qorl6.RewardText, ""),        IFNULL(qorl8.RewardText, ""),
IFNULL(qri.CompletionText, ""),     IFNULL(qril2.CompletionText, ""),    IFNULL(qril3.CompletionText, ""),   IFNULL(qril4.CompletionText, ""),    IFNULL(qril6.CompletionText, ""),    IFNULL(qril8.CompletionText, ""),
```

and added this after line 113:

```
        LEFT JOIN
            quest_offer_reward_locale qorl2 ON q.ID = qorl2.ID AND qorl2.locale = "frFR"
        LEFT JOIN
            quest_offer_reward_locale qorl3 ON q.ID = qorl3.ID AND qorl3.locale = "deDE"
        LEFT JOIN
            quest_offer_reward_locale qorl4 ON q.ID = qorl4.ID AND qorl4.locale = "zhCN"
        LEFT JOIN
            quest_offer_reward_locale qorl6 ON q.ID = qorl6.ID AND qorl6.locale = "esES"
        LEFT JOIN
            quest_offer_reward_locale qorl8 ON q.ID = qorl8.ID AND qorl8.locale = "ruRU"
        LEFT JOIN
            quest_request_items_locale qril2 ON q.ID = qril2.ID AND qril2.locale = "frFR"
        LEFT JOIN
            quest_request_items_locale qril3 ON q.ID = qril3.ID AND qril3.locale = "deDE"
        LEFT JOIN
            quest_request_items_locale qril4 ON q.ID = qril4.ID AND qril4.locale = "zhCN"
        LEFT JOIN
            quest_request_items_locale qril6 ON q.ID = qril6.ID AND qril6.locale = "esES"
        LEFT JOIN
            quest_request_items_locale qril8 ON q.ID = qril8.ID AND qril8.locale = "ruRU"
```

I hope this is useful for you.

**Comment by Sarjuuk** [comment]
fixed in 8620cbcf206fb9b92bb87c74835b591da322fcec

this is the most ridiculous query...

**Comment by TheWinchesters** [comment]
Thank you for keeping this amazing tool up to date.
I am really happy with this.
